README.md

```diff
@@ -1,13 +1,50 @@
 # Publishing API
 
-An application to route requests between multiple content-store
-endpoints based on whether they're live or in other states.
+This is a Go application that proxies requests to multiple content-stores. Our
+use case for this is to keep two copies of the frontend of GOV.UK running: one
+which the public sees and another which is only used by people working on
+content to review work in progress.
+
+Publishing apps will talk to the publishing-api rather than the content-store.
+Normal publishing requests are forwarded to both the live and draft
+content-stores, whereas draft documents would only be forwarded to the draft
+content-store.
+
+### Dependencies
+
+- [alphagov/url-arbiter](https://github.com/alphagov/url-arbiter) - ?
```

Comment:
That's deliberately blocked.  We want a publishing app to be able to create a draft of a piece of content under a path, and know that it will be allowed to publish it when the time comes.

What we will probably need at some point is an API endpoint in url-arbiter to allow a publishing app to release a path that it's no longer using.  We can deal with that when the actual need arises though.